File: lecture_002(Normal)/l_015_sumofnosdivisibleby3and5.java
import java.util.Scanner;
public class l_015_sumofnosdivisibleby3and5 {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        int a = scn.nextInt();
        int b = scn.nextInt();
        System.out.println(sumdivisible(a,b));
    }
    public static int sumdivisible(int a, int b) {
        int sum = 0;
        for(int n=a; n<=b; n++) {
            if(n%3==0 && n%5==0) {
                sum = sum+n;
            }
        }
        return sum;
    }
}

File: lecture_002(Normal)/l_015_sumofnosdivisibleby3and5.cpp
#include<iostream>
using namespace std;

int sumdivisible(int a, int b) {
    int sum = 0;
    for(int n=a; n<=b; n++) {
        if(n%3==0 && n%5==0) {
            sum = sum + n;
        }
    }
    return sum;
}
int main(int args, char**argv) {
    int a,b;
    cin>>a>>b;
    cout<<sumdivisible(a,b)<<endl;
    
    return 0;
}